neurodocker/cli/minify/tests/test_minify.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+
+from click.testing import CliRunner
+import pytest
+
+from neurodocker.cli.minify.trace import minify
+
+docker = pytest.importorskip("docker", reason="docker-py not found")
+
+
+def test_minify():
+    client = docker.from_env()
+    container = client.containers.run("python:3.9-slim", detach=True, tty=True)
```

Comment:
it's not crucial, but in the future we can create a fixture from `container` with finalizer, so the tests would not have to contain `try/except` blocks. can create a PR after this is merged

Reply:
Yes I made a context manager for this purpose. That would be in a separate PR I think. https://github.com/ReproNim/neurodocker/blob/932914b44b538129133f2a14fee1fd9db8f5c6b2/docs/test-all-templates/test_all_templates.py#L28-L54